cli/src/main/java/com/github/ruediste1/btrbck/cli/CliMain.java

```diff
@@ -509,15 +523,14 @@ private StreamRepository readAndLockRepository() {
 		if (path == null) {
 			path = Paths.get("").toFile();
 		}
-		StreamRepository repo = streamRepositoryService.readRepository(path
-				.toPath());
+		StreamRepository repo = streamRepositoryService.readRepository(path.toPath());
 		FileChannel f;
 		try {
 			f = FileChannel.open(repo.getRepositoryLockFile(),
 					StandardOpenOption.WRITE);
 			repositoryLock = f.lock(0L, Long.MAX_VALUE, false);
 		} catch (IOException e) {
-			throw new RuntimeException("Error while locking repository", e);
+			throw new DisplayException("Unable to open lock file: "+repo.getRepositoryLockFile());
```

Comment:
I'd suggest the following: if the lock file is not present, throw the DisplayException ( :+1: for using it). Otherwise, leave the try catch as is. There are more things that could go wrong, such as no permission ...